doc/telemetry-dev-env/Readme.md

```diff
@@ -0,0 +1,47 @@
+Goto doc/telemetry-dev-env
+Build the image:
+```
+docker build -t telemetry-dev .
+```
+
+Remove container if it exist already:
+```
+docker rm telemetry-dev-env
+```
+Create personal fork repo from sonic-telemetry from git
+
+Create directory '~/src' and Clone git repos in it:
+```
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-telemetry.git 
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-mgmt-common.git
+```
+
+Create a new container using the image:
+```
+docker run -it -v ~/src:/src --net=host --name telemetry-dev-env telemetry-dev
+```
+
+Build and start service (Inside container):
+```bash
+
+# Build sources
+cd /src/sonic-mgmt-common && make
+cd /src/sonic-telemetry && make all
+
+# Bootstrap DB.
+mkdir -p /var/run/redis/sonic-db && chown redis  /var/run/redis/sonic-db && install testdata/database_config.json -t /var/run/redis/sonic-db
+service redis-server stop && cp /files/dump.rdb /var/lib/redis/dump.rdb && service redis-server start
```

Comment:
is it an empty file? why it is required if it is empty?

Reply:
idea is developer can replace this rdb with own contents if they want to see any output of path. Although empty rdb wont yield anything but listing step is needed

Reply:
you could add this info into comment line after "Bootstrap DB."